test/load/allocation/grpc/README.md

```diff
@@ -0,0 +1,70 @@
+# Load Test for gRPC allocation service
+
+This load tests aims to validate performance of the gRPC allocation service.
+
+## Kubernetes Cluster Setup
+
+For the load test you can follow the regular Kubernetes and Agones setup. In order to test the allocation performance in isolation, we let Agones to get all
+the game servers to Ready state before starting a test.
+Here are the few important things:
+- If you are running in GCP, use a regional cluster instead of a zonal cluster to ensure high availability of the cluster control plane
+- Use a dedicated node pool for the Agones controllers with multiple CPUs per node, e.g. `n1-standard-4'
+- In the default node pool (where the Game Server pods are created), 75 nodes are required to make sure there are enough nodes available for all game servers to move into the ready state. When using a regional cluster, with three zones with the region, that will require a configuration of 25 nodes per zone.
+
+## Fleet Setting
+
+We used the sample [fleet configuration](https://github.com/googleforgames/agones/blob/master/examples/fleet.yaml) with some minor modifications. We updated the `replicas` to 4000. 
+Also w set the `automaticShutdownDelayMin` parameter to 10 so simple-udp game servers shutdown after 10 minutes (see below).
+This helps to easily re-run the test without having to delete the game servers and allows to run tests continously. 
+
+```yaml
+apiVersion: "agones.dev/v1"
+kind: Fleet
+ ...
+ spec:
+  # the number of GameServers to keep Ready 
+  replicas: 4000
+  ...
+        # The GameServer's Pod template
+        template:
+            spec:
+                containers:
+                - args:
+                  # We setup the simple-udp server to shutdown 10 mins after allocation 
+                  - -automaticShutdownDelayMin=10
+                  image: gcr.io/agones-images/udp-server:0.21
+                  name: simple-udp
+  ...
+```
+
+## Configuring the Allocator Service
+
+The allocator service uses gRPC. In order to be able to call the service, TLS and mTLS has to be setup.
+For more information visit [Allocator Service](https://agones.dev/site/docs/advanced/allocator-service).
+
+## Running the test
+
+You can use the provided runAllocation.sh script by providing two parameters: 
+- number of clients (to do parallel allocations)
+- number of allocations for client
+
+For making 4000 allocations calls, you can provide 40 and 100
+
+```
+./runAllocation.sh 40 100
+```
+
```

Comment:
Thinking we might want to also make mention of the environment variables that are able to be set, as well as their default values, an example might also be nice, something like:

To run only a single run of tests, you can use: 

```bash
$ TESTRUNSCOUNT=1 ./runAllocation.sh 40 10
```

Reply:
Add a sample